.github/workflows/ci.yaml

```diff
@@ -62,6 +62,273 @@ jobs:
           cache-from: type=gha
           cache-to: type=gha,mode=max
 
+  deploy-test-sp3-rpm:
```

Comment:
Doesn't look this as a perfect place to use a `matrix`?
I mean, the unique difference is the IP address as far as I see